alacritty/src/config/mod.rs

```diff
@@ -313,11 +313,12 @@ fn print_deprecation_warnings(config: &Config) {
         )
     }
 
-    #[cfg(windows)]
+    #[cfg(all(windows, not(feature = "winpty")))]
     if config.winpty_backend {
         warn!(
             target: LOG_TARGET_CONFIG,
-            "Config winpty_backend is deprecated; please use the ConPTY backend instead",
+            "Config winpty_backend is deprecated and requires a compilation flag; please use the \
```

Comment:
In that form it's not like you should use ConPTY backend, it's like you have no choice and the only thing you can do is to remove it, since you're already using ConPTy backend. So, I'd removed the last part of that sentence, or reworded it.

Reply:
How would you reword it? We want the user to use the ConPTY backend, instead of recompiling with WinPTY. So that's why I've worded it this way.

Reply:
Maybe something like that?

 "Config winpty_backend is deprecated and requires a compilation flag; it can be removed from the config"

Reply:
But at that point, doesn't it sound like the compilation flag should be enough? Which it is not, you'd still need the configuration option.

Reply:
I just don't like `Please use ConPTY`, because you can't really not use it.

Reply:
I think it should be fine now. If someone can compile it from source, they should be able to figure out that the option is still required.